app/ChatUser.php

```diff
@@ -19,8 +19,50 @@ public function sender()
         return $this->belongsTo(User::class, 'sender_id');
     }
 
+    public static function lastCommunicationDate($id_1, $id_2)
+    {
+        return self::where(function ($query) use ($id_1, $id_2) {
+            return $query->where('reciever_id', '=', $id_1)
+                ->where('sender_id', '=', $id_2);
+        }
+        )->orWhere(function ($query) use ($id_1, $id_2) {
+            return $query->where('reciever_id', '=', $id_2)
+                ->where('sender_id', '=', $id_1);
+        }
+        )->orderBy('updated_at', 'desc')
+            ->first()->updated_at;
```

Comment:
Yes but edition != last comment on the page. for example
```
James: 01.01.01 Hellow how are you ?
Nicolas: 02.01.01 Hi, I'm fine.
```
Now James see that he has wrongly spelled hello and do the change

```
Nicolas: 02.01.01 Hi, I'm fine.
James: 01.01.01 Hello how are you ?
```

I don't think it's the behaviour you expected in a chat